code/modules/paperwork/pen.dm

```diff
@@ -24,6 +24,9 @@
 	var/colour = "black"	//what colour the ink is!
 	pressure_resistance = 2
 
+/obj/item/weapon/pen/suicide_act(mob/user)
+		viewers(user) << "<span class='suicide'>[user] starts scribbling over \himself with the [src.name]! It looks like \he's trying to commit sudoku.</span>"
```

Comment:
Yeah, these are tabs. It wouldn't compile otherwise, BYOND is pretty finicky about whitespace.